airflow/operators/druid_check_operator.py

```diff
@@ -22,7 +22,7 @@
 from airflow.providers.apache.druid.operators.druid_check import DruidCheckOperator  # noqa
 
 warnings.warn(
-    "This module is deprecated. Please use `airflow.operators.sql.SQLCheckOperator`.",
+    "This module is deprecated. Please use `airflow.providers.apache.druid.operators.druid_check` module.",
```

Comment:
```suggestion
    "This module is deprecated. Please use `airflow.providers.common.operators.sql` module.",
```
Looks like the `DruidCheckOperator` and the module is also deprecated and points to the `SQLCheckOperator`. Deprecation inception.

Reply:
Ah. That's correct we did not need DruidCheck - plain SQLcheck was enough passing Druid Hook.  So all is good after my corrections.